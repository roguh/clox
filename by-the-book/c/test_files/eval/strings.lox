print("Hello, world!!!");
print("He" + "llo," + " W" + "orld" + "!");
print(#"Hello, World!");
