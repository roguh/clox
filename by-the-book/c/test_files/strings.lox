"abcdef" '123' "123456\""
