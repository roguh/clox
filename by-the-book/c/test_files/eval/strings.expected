Hello, world!!!
Hello, World!
13
